src/Events/QR/Routes.php

```diff
@@ -75,7 +76,7 @@ public function do_register(): void {
 		 */
 		$this->route_prefix = apply_filters( 'tec_events_qr_route_prefix', 'qr' );
```

Comment:
Same here, dont filter this early, you dont give a chance for people to hook in.

Create a method that returns the route prefix and filter it.